### ISSUE TITLE
{PROJECT} is not replaced

### ISSUE THREAD

**Issue by onmyway133** [JohnSundell/SwiftPlate#39]
Hi, 

I'm running through SwiftPlate, on the last step of generating the project, I see that the {PROJECT} placeholder is not being updated. This is the screenshot

![swift](https://cloud.githubusercontent.com/assets/2284279/22905109/4c0e1468-f23f-11e6-8df7-c97f0ae05616.png)

This is my terminal

```sh
./main.swift
Welcome to the SwiftPlate project generator 🐣
📦  Where would you like to generate a project? (Leave empty to use current directory)
/Users/khoa/Downloads/Superman
📛  What's the name of your project? (Leave empty to use the name of the project folder: Superman)

👶  What's your name? (Leave empty to use your git config name: Khoa Pham)

📫  What's your email address (for Podspec)? (Leave empty to use your git config email: onmyway133@gmail.com)

🌍  Any GitHub URL that you'll be hosting this project at (for Podspec)? You may leave this empty.

🏢  What's your organization name? You may leave this empty.

---------------------------------------------------------------------
SwiftPlate will now generate a project with the following parameters:
📦  Destination: /Users/khoa/Downloads/Superman
📛  Name: Superman
👶  Author: Khoa Pham
📫  Author email: onmyway133@gmail.com
---------------------------------------------------------------------
Proceed? ✅ (Y/N)
y
🚀  Starting to generate project Superman...
👉  Removing any previous temporary folder...
✅  Done
👉  Making temporary folder (/Users/khoa/Downloads/Superman/swiftplate_temp)...
✅  Done
👉  Making a local clone of the SwiftPlate repo...
✅  Done
👉  Copying template folder...
✅  Done
👉  Removing temporary folder...
✅  Done
👉  Filling in template...
✅  Done
All done! 🎉  Good luck with your project! 🚀
```

I expect the `{PROJECT}` placeholder be replaced by `Superman`, maybe I miss something 🤔

**Comment by vijaytholpadi** [comment]
Fixed in #42

**Comment by onmyway133** [comment]
@vijaytholpadi thanks 🚀
